Build broken and not reporting failure

On OSX: https://travis-ci.org/twall/jna/jobs/69478093#L636

```
   [javac] Compiling 66 source files to /home/travis/build/twall/jna/contrib/platform/build/test/classes
    [javac] warning: [options] bootstrap class path not set in conjunction with -source 1.5
    [javac] /home/travis/build/twall/jna/contrib/platform/test/com/sun/jna/platform/mac/SystemBTest.java:20: error: package oshi.software.os.mac.local.SystemB does not exist
    [javac] import oshi.software.os.mac.local.SystemB.HostCpuLoadInfo;
    [javac]                                          ^
    [javac] /home/travis/build/twall/jna/contrib/platform/test/com/sun/jna/platform/mac/SystemBTest.java:21: error: package oshi.software.os.mac.local.SystemB does not exist
    [javac] import oshi.software.os.mac.local.SystemB.HostLoadInfo;
    [javac]                                          ^
    [javac] /home/travis/build/twall/jna/contrib/platform/test/com/sun/jna/platform/mac/SystemBTest.java:22: error: package oshi.software.os.mac.local.SystemB does not exist
    [javac] import oshi.software.os.mac.local.SystemB.VMStatistics;
    [javac]                                          ^
    [javac] /home/travis/build/twall/jna/contrib/platform/test/com/sun/jna/platform/mac/SystemBTest.java:23: error: package oshi.software.os.mac.local.SystemB does not exist
    [javac] import oshi.software.os.mac.local.SystemB.VMStatistics64;
    [javac]                                          ^
    [javac] /home/travis/build/twall/jna/contrib/platform/test/com/sun/jna/platform/mac/SystemBTest.java:89: error: cannot find symbol
    [javac] 		VMStatistics vmStats = new VMStatistics();
    [javac] 		^
```

--- a/.travis.yml
+++ b/.travis.yml
@@ -6,7 +6,7 @@ install:
 
 script:
   - ant test
-  - cd contrib/platform ; ant test ; cd ../..
+  - ant -f ./contrib/platform/build.xml test
   - ant dist
 
 os:

--- a/.travis.yml
+++ b/.travis.yml
@@ -6,7 +6,7 @@ install:
 
 script:
   - ant test
-  - ant -f ./contrib/platform/build.xml test
+  - ant test-platform
   - ant dist
 
 os:

